pages/playlist/curated/[id]/index.vue

```diff
@@ -10,15 +10,16 @@ const { data: playlist } = useCuratedPlaylist({ id: playlistId });
 const { data: tracks, pending } = useCuratedPlaylistTracks({ id: playlistId });
 
 const { setQueueShuffled, setQueue } = useNuxtApp().$mediaPlayer;
+const origin = () => `CuratedPlaylist|${playlistId}|${playlist.value?.title}`;
```

Comment:
These could probably be `computed` values, so Vue can optimize things better.

Reply:
When using a `computed`, Vue will only re-run the getter if any reactive values inside it are updated. 
In this case it will only re-run when `playlist.value.title` is updated, and otherwise just return the previously computed value

Reply:
done